### ISSUE TITLE
No confirmation is announced by the narrator after activating 'Remove equation' button

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1386]
**Pre-Requisites:**
Expressions should be added.

**Repro Steps:** 
1. Launch Calculator app.
2. Turn on the narrator.
3. Navigate Open navigation button and activate it.
4. Navigate to graphing calculator list item and activate it.
5. Navigate to Enter a expression edit box.
6. Enter any expression say "7+8".
7. Now open context menu and activate "Remove equation" button.
8. Observe the issue.

**Actual Result:**
No confirmation is announced by the narrator after activating 'Remove equation' button.

**Expected Result:**
Confirmation should be announced by the narrator after activating 'Remove equation' button.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
